include/exadg/incompressible_navier_stokes/user_interface/enum_types.h

```diff
@@ -359,14 +359,7 @@ enum class QuadratureRuleLinearization
  *  use CG (conjugate gradient) method as default. FGMRES might be necessary
  *  if a Krylov method is used inside the preconditioner (e.g., as multigrid
  *  smoother or as multigrid coarse grid solver)
- */
```

Comment:
note that all of these solvers support now the solver supported in `dealii::SolverSelector` automatically.